consumer/pdata/metric_test.go

```diff
@@ -18,9 +18,9 @@ import (
 	"testing"
 
 	gogoproto "github.com/gogo/protobuf/proto"
-	goproto "github.com/golang/protobuf/proto" //lint:ignore SA1019 golang/protobuf/proto is deprecated
-	otlpmetricsgoproto "github.com/open-telemetry/opentelemetry-proto/gen/go/metrics/v1"
```

Comment:
Is this change needed to stop importing `github.com/open-telemetry/opentelemetry-proto/gen` (which no longer exists)

Reply:
Yes, remove the warning